src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -683,17 +683,25 @@ namespace static_llm {
 StatefulLLMPipeline::StatefulLLMPipeline(
     const std::filesystem::path& models_path,
     const ov::genai::Tokenizer& tokenizer,
-    const std::string&,
+    const std::string& device,
     const ov::AnyMap& config
 ) : LLMPipelineImplBase(tokenizer,
                         utils::from_config_json_if_exists(models_path)) {
+    if (std::filesystem::exists(models_path / "openvino_model.blob")) {
+        std::ifstream fin(models_path / "openvino_model.blob", std::ios::in | std::ios::binary);
+        ov::AnyMap properties = config;
+        ModelConfigDesc model_desc = get_modeldesc_from_json(models_path / "config.json");
+        updateStatefulConfig(model_desc, properties);
```

Comment:
This info should be already embedded to the blob

Reply:
Done